widget/widget.css

```diff
@@ -190,19 +190,22 @@ label {
 
 .model-details {
   width: 700px;
-  border-style: solid;
-  border-width: 1px;
+  margin-top: 0px !important;
+  border-style: solid !important;
+  border-width: 1px !important;
 }
 
 .model-details-row {
   width: 100%;
   display: inline-block;
-  text-align: center;
+  text-align: center !important;
+  padding: 0.1em 0.1em !important;
```

Comment:
@Nking92 It would be nice if you could avoid using the `!important` directive.
It is generally considered bad practice [see here](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity#:~:text=Using%20!,greater%20specificity%20will%20be%20applied.).

Reply:
Fixed with some more specific selectors